packages/notifi-react-card/lib/components/intercom/ChatMessageDate.tsx

```diff
@@ -29,13 +30,20 @@ export const ChatMessageDate: React.FC<ChatMessageDateProps> = ({
     >
       <div
         className={clsx(
-          'NotifiIntercomChatMessageDate__content',
-          classNames?.content,
+          'NotifiIntercomChatMessageDate__body',
+          classNames?.body,
         )}
       >
-        {isStartDate
-          ? formatConversationStartTimestamp(createdDate)
-          : formatConversationDateTimestamp(createdDate)}
+        <div
+          className={clsx(
+            'NotifiIntercomChatMessageDate__content',
+            classNames?.content,
+          )}
+        >
```

Comment:
a div with only one div as a child feels unnecessary, can you see if combining the styles for these two classnames solves your problem?

Reply:
hmm, it doesn't work when combine the styles, it should work with inline-block, and width with 100%, but as the initial div container is the container with background of the date component, so I can't make width equals to 100%.